app/controllers/application_controller/timelines.rb

```diff
@@ -127,41 +127,12 @@ def tl_chooser
     end
   end
 
-  def getTLdata
-    if session[:tl_xml_blob_id] != nil
-      blob = BinaryBlob.find(session[:tl_xml_blob_id])
-      render :xml=>blob.binary
+  def timeline_data
```

Comment:
Indeed. Though, as it's apparently only used by three routes, I think I'd rather reduce the massive ApplicationController surface area by one method, and move it to whichever third controller is currently actually capable of getting here.